packages/core/src/new-api/deploy.ts

```diff
@@ -14,13 +14,15 @@ import { IgnitionModuleDefinition } from "./types/module-builder";
  * @beta
  */
 export async function deploy({
+  config,
   artifactResolver,
   adapters,
   deploymentDir,
   moduleDefinition,
   deploymentParameters,
   accounts,
 }: {
+  config?: Partial<DeployConfig>;
```

Comment:
This is the test helper function, I wanted the option to change config setting (specifically the timeout for transactions)